### PR TITLE
[jsk_tools] Add rossetdefault, rosdefault to zshrc.ros

### PR DESCRIPTION
**Pull request by wkentaro**
Usage
-----
```
user@host $ rossetdefault pr1040
set ROS_MASTER_URI to http://pr1040:11311
[http://pr1040:11311] user@host $
user@host $ rosdefault
set ROS_MASTER_URI to http://pr1040:11311
[http://pr1040:11311] user@host $
user@host $ rossetdefault local
user@host $ rosdefault
set ROS_MASTER_URI to http://localhost:11311
```
It is recommended to write `rosdefault` in your .bashrc or .zshrc.

### CODE REVIEW

**Comment by k-okada** [comment]
please take care of both bash and zsh settings, using different command
between zsh and bash users may confusing.

--
◉ Kei Okada

**Comment by wkentaro** [comment]
okay I will also add this function to bashrc.ros.

**Comment by garaemon** [comment]
I think there is a lot of similar function between zsh and bash, Can we use same file?
for example add `source $(rospack find jsk_tools)/common.sh`

**Comment by wkentaro** [comment]
I think it is possible and not so hard.
I opened issue #869

File: jsk_tools/env-hooks/99.jsk_tools.bash [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
 # -*- mode: Shell-script; -*-
 
+function rossetdefault() {
+    local ros_master_default=${1-"local"}
+    echo $ros_master_default > ~/.rosdefault
+    rosdefault
+}
+function rosdefault()
+{
+    local ros_master_default=`cat ~/.rosdefault`
```
Comment (garaemon) [inline jsk_tools/env-hooks/99.jsk_tools.bash]: Check if ~/.rosdefault exists

**Comment by wkentaro** [comment]
I updated.

**Comment by garaemon** [comment]
I'm afrait `rossetdefault` may be too complicated utility but I merge it in order to check if it's useful.

**Comment by wkentaro** [comment]
I see.

It may become simple command by thinking inversely, 

`rosdefault` is the command set `ROS_MASTER_URI` with values written in `~/.rosdefault`.

And

`rossetdefault` is the command by which you can easily rewrite `~/.rosdefault`.

**Comment by k-okada** [comment]
it think this kind of files should go to ~/.ros ($ROS_HOME)http://wiki.ros.org/ROS/EnvironmentVariables#ROS_HOME

**Pull request merged**